<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="CommandFrame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #31363b;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="TransactionButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Futura</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <property name="text">
           <string>Transaction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CloseSessionButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
          </property>
          <property name="text">
           <string>Close Session</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="HoldButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Hold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PaymentButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Payment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SyncButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Sync</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Session</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>--</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>--</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ProductButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Product</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PrintButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Print</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ExitButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Futura</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    color: #ffff00;
    background-color: #31363b;
    border-width: 1px;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color : #31363b;
	background-color:#ffff00;
    border-width: 1px;
    border-color: : #ffff00;
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
            </property>
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
